lib/build.gradle

```diff
@@ -1,17 +1,15 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
-apply plugin: 'com.novoda.bintray-release'
+//apply plugin: 'com.novoda.bintray-release'
```

Comment:
How to publish the update in this case?

Reply:
I personally do use jitpack.io to publish github projects, I never tried bintray yet...

Reply:
It is much easier with bintray. The older versions are published there. Please uncomment this and publish section

Reply:
did this already...

Reply:
thanks